test/Reader/ReaderTest.php

```diff
@@ -309,6 +319,37 @@ public function testImportStringMethodThrowProperExceptionOnEmptyString()
         $feed = Reader\Reader::importString($string);
     }
 
+    public function testSetHttpFeedClient()
```

Comment:
Both setHttpClient tests could be refactored as a single one and use PHPUnit 4.8 `testWith` https://github.com/sebastianbergmann/phpunit/pull/1728/files#diff-bb228c7df86bfe5aafe50375a8d62784R5 (if use this feature please update composer.json too)